src/test/java/cz/czechitas/AdminShopPage.java

```diff
@@ -0,0 +1,51 @@
+package cz.czechitas;
+
+import java.util.*;
+import org.openqa.selenium.*;
+import org.openqa.selenium.support.ui.*;
+import okhttp3.internal.http2.*;
+
+public class AdminShopPage {
+
+
+    WebDriver driver;
+   
+
+
+    public AdminShopPage(WebDriver driver) {
+        this.driver = driver;
+    }
+
+
+    public void openAdminpage() {
+        driver.navigate().to(Settings.adminLogonUrl);
+    }
+
+    public void fillLoginDetails (String username, String password) {
+        WebElement usernameField = driver.findElement(By.id("username"));
+        usernameField.sendKeys(username);
+        WebElement passwordField = driver.findElement(By.id("password"));
+        passwordField.sendKeys(password);
+    }
+
+    public void logonClick() {
+        WebElement logonField = driver.findElement(By.id("formSubmitButton"));
+        logonField.click();
+    }
+
+
+    public String getErrorMessage () {
+        WebElement errorMessageField = driver.findElement(By.className("alert"));
+        return errorMessageField.getText();
+    }
+
+    public void waitForLinkText (String linkText) {
```

Comment:
Toto neni uplne zle, ale je to uz hodne obecne. Page objecty by mely byt spis specificke. Ja osobne bych problem overeni, ze jsem zalogovana asi resila vytvorenim dalsiho Page Objectu, ktery bude reprezentovat stranku pote, co se uzivatel zaloguje a bude kontrolovat neco konkretniho co se na te strance vyskytuje. Nicmene takovehle reseni taky muzu akceptovat.